core/modules/layout/layout.admin.inc

```diff
@@ -795,6 +795,26 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     );
   }
 
+  if (isset($form_state['build_info']['args'][0]->settings['additional_paths'])) {
```

Comment:
This could use a comment to explain why `$layout->settings['additional_paths']` wouldn't work here.

Reply:
I think I just copied this from the contrib module. `$layout->settings['additional_paths']` seems to work fine too.